app/models/group.rb

```diff
@@ -4,6 +4,7 @@ class Group < StudyRelationship
   has_many :outcomes
   has_many :outcome_measures
   has_many :outcome_analyses
+  has_many :reported_results
```

Comment:
Yes!  Thanks.


________________________________
From: Garrett Martin <notifications@github.com>
Sent: Wednesday, August 10, 2016 11:23:39 AM
To: SturdyCo/aact2
Cc: Sheri Tibbs; Author
Subject: Re: [SturdyCo/aact2] Aact 134 limitations and caveats (#57)


In app/models/group.rb<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_SturdyCo_aact2_pull_57-23discussion-5Fr74266556&d=CwMCaQ&c=imBPVzF25OnBgGmVOlcsiEgHoG1i6YHLR0Sj_gZ4adc&r=UHmIZTL_6Fe-uBOlhrUpgwa0k8zeM9De7MnTpJOTolE&m=sAMVW-Pt4Yz34_FpPWe7Qqzlzgq2aAykphDfr_ltBBY&s=Wqh9ntN0H_xYl_5O807axtwuGoFZ3aPWNASWO6aZHQQ&e=>:

> @@ -4,6 +4,7 @@ class Group < StudyRelationship
>    has_many :outcomes
>    has_many :outcome_measures
>    has_many :outcome_analyses
> +  has_many :reported_results


Should this be reported_events?

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_SturdyCo_aact2_pull_57_files_0b787af8b7f507e94a2f8f1c347e050bbc130a57-23r74266556&d=CwMCaQ&c=imBPVzF25OnBgGmVOlcsiEgHoG1i6YHLR0Sj_gZ4adc&r=UHmIZTL_6Fe-uBOlhrUpgwa0k8zeM9De7MnTpJOTolE&m=sAMVW-Pt4Yz34_FpPWe7Qqzlzgq2aAykphDfr_ltBBY&s=WJ-YSqDtQSGbVA28jd1TMSNUTNo95NRIFLJY0Sz_XTo&e=>, or mute the thread<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_ABLfK5CovugAsYfaNsH7aqVZwz7cD1o0ks5qeez7gaJpZM4Jgv-5FB&d=CwMCaQ&c=imBPVzF25OnBgGmVOlcsiEgHoG1i6YHLR0Sj_gZ4adc&r=UHmIZTL_6Fe-uBOlhrUpgwa0k8zeM9De7MnTpJOTolE&m=sAMVW-Pt4Yz34_FpPWe7Qqzlzgq2aAykphDfr_ltBBY&s=wRgGx6tl_33tYxIWA5NXA7d7zctEMh8qQP4_BXXDd3o&e=>.